### PR TITLE
Disable invalid certificate test setup

### PR DESCRIPTION
**Pull request by abhipsaMisra**

### CODE REVIEW

File: build.ps1 [hunk 1]
```diff
@@ -395,6 +395,7 @@ try
         # test categories to exclude
         $testCategory += "&TestCategory!=LongRunning"
         $testCategory += "&TestCategory!=Flaky"
+        $testCategory += "&TestCategory!=InvalidServiceCertificate"
```
Comment (drwill-ms) [inline build.ps1:398]: Add a comment to indicate this is temporary until the pipeline is fixed for docker commands on Linux?

Can we only disable it for Linux?

File: vsts/vsts.yaml [hunk 2]
```diff
@@ -167,84 +167,84 @@ jobs:
...
-            cat /etc/hosts
-            echo "====================="
-            echo "Ping containers (URL)"
-            echo "====================="
-            docker ps -a
-            route
-            ping -c 2 invalidcertgde1.westus.cloudapp.azure.com
-            ping -c 2 invalidcertdps1.westus.cloudapp.azure.com
-            ping -c 2 invalidcertiothub1.westus.cloudapp.azure.com
+      # - task: Bash@3
```
Comment (drwill-ms) [inline vsts/vsts.yaml:170]: Same, add comment why.

Comment (abhipsaMisra) [reply vsts/vsts.yaml:170]: Sure

File: build.ps1 (hunk 1 above)
Comment (abhipsaMisra) [reply build.ps1:398]: sure

**Pull request merged**